### PR TITLE
Update Symbols.md

### PR DESCRIPTION
**Pull request by nblyumberg**
Added a note regarding glob matching as a bookmark for further explanation of behavior or until this can be made generic to work on *.xml or similar files.

### CODE REVIEW

File: docs/Symbols.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ The order of the expression item are important, because each expression are appl
 
 NOTE: when you change `xml.symbols.filters` in `settings.json`, the Outline is not refreshed automatically (see vscode [vscode issue 108722](https://github.com/microsoft/vscode/issues/108722)). You must refresh at hand the Outline by updating the XML file or closing/opening the XML file.
 
+NOTE: when you define your own patterns in `settings.json`, it might be necessary to define the glob pattern that crosses directory boundries `**` for the matching to occur. ([vscode-xml issue 684](https://github.com/redhat-developer/vscode-xml/issues/684)).
```
Comment (fbricon) [inline docs/Symbols.md:67]: boundaries

Comment (fbricon) [reply docs/Symbols.md:67]: You should add an example

Comment (nblyumberg) [reply docs/Symbols.md]: Ok I think I did it the right way with the Github web UI

Comment (fbricon) [reply docs/Symbols.md]: still needs to be spelled "boundaries" ;-)

File: docs/Symbols.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ The order of the expression item are important, because each expression are appl
 
 NOTE: when you change `xml.symbols.filters` in `settings.json`, the Outline is not refreshed automatically (see vscode [vscode issue 108722](https://github.com/microsoft/vscode/issues/108722)). You must refresh at hand the Outline by updating the XML file or closing/opening the XML file.
 
+NOTE: when you define your own patterns in `settings.json`, it might be necessary to define the glob pattern that crosses directory boundaries `**` for the matching to occur. ([vscode-xml issue 684](https://github.com/redhat-developer/vscode-xml/issues/684)). For example, when applying this to xml and bpml files this might be necessary: `"pattern": "**/*.{bpml,xml}",`
```
Comment (AlexXuChen) [inline docs/Symbols.md:67]: Capitalize XML and BPML for consistency.

Comment (AlexXuChen) [inline docs/Symbols.md:67]: "(See `[vscode-xml issue 684](https://github.com/redhat-developer/vscode-xml/issues/684)` for more detail.)."

**Review by AlexXuChen — CHANGES_REQUESTED** [review]

File: docs/Symbols.md (hunk 2 above)
Comment (nblyumberg) [reply docs/Symbols.md]: I removed the ( ) since it no longer made sense to have them around a full sentence.

**Review by AlexXuChen — APPROVED** [review]
LGTM

**Comment by AlexXuChen** [comment]
Thanks @nblyumberg!

**Pull request merged**